linear_search/linear_search.js

```diff
@@ -0,0 +1,29 @@
+function linearSearch (arr, x) {
+	/*
+	Performs a linear search
+	:param arr: List of elements to search from
+	:param x: Element to search for
+	:return: Index if element found else -1
+	*/
+	for (let it in arr) {
+		if (arr[it] === x) {
+			return it;
+		}
+	}
+	return -1;
+}
+
+function main () {
+	let A = [];
+	for (let i = 1; i<=100; i++) {
```

Comment:
In line 18 
`for (let i = 1; i<=100; i++) {`
change it to 
`for (let i = 1; i <= 100; i++) {`